.github/workflows/ci.yml

```diff
@@ -1,8 +1,5 @@
 name: CI
 on:
-  push:
-    branches:
-      - main
   pull_request:
```

Comment:
Could we add a workflow_dispatch too? To be able to run tests/ci for main if needed.

Reply:
We can check it in the release previously to release the jar